docs/spec/glossary.rst

```diff
@@ -38,10 +124,37 @@ This section defines a few terms that may be used elsewhere in the specification
       be imported from the :py:mod:`typing` module or equivalently from ``typing_extensions``,
       but some special forms are placed in other modules.
 
+   structural
+      A structural type (see e.g. :ref:`Protocols`, :ref:`TypedDict`) defines a
+      set of values not by their ``__class__``, but by their properties (e.g.
+      attributes, methods, dictionary key/value types). :ref:`Callable` types
+      are also structural; a callable type is a subtype of another callable
+      type based on their signatures, not a subclass relationship. In contrast,
+      see :term:`nominal` types.
+
    stub
       A file containing only type information, empty of runtime code
       (the filename ends in ``.pyi``). See :ref:`stub-files`.
 
+   subtype
+      A :term:`fully static type` ``B`` is a subtype of a fully static type
+      ``A`` if and only if the set of possible runtime values represented by
+      ``B`` is a subset of the set of possible runtime values represented by
+      ``A``. For :term:`nominal` types (classes), subtyping is defined by
+      inheritance. For :term:`structural` types, subtyping is defined by a
+      shared set of attributes/methods or keys. Subtype is the inverse of
+      :term:`supertype`. Types that are not fully static are not a subtype or
+      supertype of any other type, but via :term:`materialization
+      <materialize>` they can be :term:`assignable` to another type. See
+      :ref:`type-system-concepts`.
```

Comment:
Either

```suggestion
      :term:`supertype`. A type that is not fully static is not a subtype or
      supertype of any other type, but via :term:`materialization
      <materialize>` can be :term:`assignable` to another type. See
      :ref:`type-system-concepts`.
```

or

```suggestion
      :term:`supertype`. Types that are not fully static are not subtypes or
      supertypes of any other types, but via :term:`materialization
      <materialize>` they can be :term:`assignable` to other types. See
      :ref:`type-system-concepts`.
```

So that you're consistent in whether you're using singular or plural